chart/values.schema.json

```
@@ -3751,11 +3751,207 @@
+                            "labels": {
+                                "dag_id": "$1",
+                                "task_id": "$2"
+                            }
+                        },
+                        {
+                            "match": "airflow.dagrun.duration.success.*",
+                            "name": "airflow_dagrun_duration",
+                            "labels": {
+                                "dag_id": "$1"
+                            }
+                        },
+                        {
+                            "match": "airflow.dagrun.duration.failed.*",
+                            "name": "airflow_dagrun_failed",
+                            "labels": {
+                                "dag_id": "$1"
+                            }
+                        },
+                        {
+                            "match": "airflow.dagrun.schedule_delay.*",
+                            "name": "airflow_dagrun_schedule_delay",
+                            "labels": {
+                                "dag_id": "$1"
+                            }
+                        },
+                        {
+                            "match": "airflow.dag_processing.last_runtime.*",
+                            "name": "airflow_dag_processing_last_runtime",
+                            "labels": {
+                                "dag_file": "$1"
+                            }
+                        },
+                        {
+                            "match": "airflow.dag_processing.last_run.seconds_ago.*",
+                            "name": "airflow_dag_processing_last_run_seconds_ago",
+                            "labels": {
+                                "dag_file": "$1"
+                            }
+                        },
+                        {
+                            "match": "airflow.pool.open_slots.*",
+                            "name": "airflow_pool_open_slots",
+                            "labels": {
+                                "pool": "$1"
+                            }
+                        },
+                        {
+                            "match": "airflow.pool.used_slots.*",
+                            "name": "airflow_pool_used_slots",
+                            "labels": {
+                                "pool": "$1"
+                            }
+                        },
+                        {
+                            "match": "airflow.pool.starving_tasks.*",
+                            "name": "airflow_pool_starving_tasks",
+                            "labels": {
+                                "pool": "$1"
+                            }
+                        }
+                    ],
+                    "items": {
+                        "type": "object",
+                        "description": "Prometheus statsd_exporter mapping configuration",
+                        "properties": {
```

```suggestion
                        "additionalProperties": false,
                        "properties": {
```

Probably want this as well?

---

Thanks, I'm still trying to figure out the exact syntax/use of these validators. I'm attempting to run some of the presubmit scripts, but since I don't work on this every day it's probably not setup correctly.

---

@SuperQ - BTW. these are standard JSON-Schema validations. IntelliJ for one (And I guess VScode and others) have great support for those and provides all the autocompletes and what-nots (which I found extremely useful to add validations without learning the whole of it).

---

Yea, things I don't use :)

---

Maybe you should :P . I also STILL use vim when I want to do something quick - but IntelliJ is something I happily pay for for years and I get Yearly subscription in ~ 1 day back in form of saved hours

---

I spend most of my time contributing to random projects. Mostly Go code. So this is a bit out of the ordinary for my workflows.